src/main/java/com/google/opensesame/util/RateLimitExceededException.java

```diff
@@ -0,0 +1,23 @@
+package com.google.opensesame.util;
+
+import java.io.IOException;
+
+/**
+ * The exception to be thrown when the GitHub API has run out of calls. This will help us avoid
+ * lengthy stalls when we lose access to GitHub Data i.e. Our API calls have exceeded our current
+ * GitHub hourly call rate limit.
+ *
+ * <p>NOTE: When this Exception is caught, the appropriate error code to throw would be an
+ * HTML_FORBIDDEN (403). This is to stay consistent with the GitHub Restful API convention found <a
+ * href="https://developer.github.com/v3/#rate-limiting">here</a>
+ */
+public class RateLimitExceededException extends IOException {
+
+  /** The default error message to be sent when OpenSesame runs out of API calls. */
+  private static String defaultErrorMessage =
+      "OpenSesame has Exceeded it's " + "available GitHub API calls";
```

Comment:
Consider moving this into the no-arg constructor below, as it doesn't provide much extra functionality or information by being in a constant like this. Take a look at these comments I got from Joe, which were quite helpful for me:

https://github.com/googleinterns/open-sesame/pull/151#discussion_r463924612
https://github.com/googleinterns/open-sesame/pull/151#discussion_r463926351

Reply:
Solid point!

Reply:
It turns out Java constructors don't allow you to make any declarations before calling the super class constructor. Tbh, the text has minimal utility. should I remove it?